pkg/override/override.go

```diff
@@ -24,9 +24,10 @@ func GetOverriddenResultName(resultName string) string {
 
 // NewRedundantOverrideResult creates a new AuditResult at warning level telling the user to remove the override
 // label because there are no security issues found, so the label is redundant
-func NewRedundantOverrideResult(containerName string, overrideReason, overrideLabel string) *kubeaudit.AuditResult {
+func NewRedundantOverrideResult(auditorName, containerName, overrideReason, overrideLabel string) *kubeaudit.AuditResult {
 	return &kubeaudit.AuditResult{
-		Name:     kubeaudit.RedundantAuditorOverride,
+		Auditor:  auditorName,
```

Comment:
Do we also need the manifest path here?

Reply:
also for the purpose of running it with github code analysis, info/note are not displayed. It just displays errors and warnings...